Library/Homebrew/cmd/test-bot.rb

```diff
@@ -209,19 +209,16 @@ def initialize(argument, options={})
       @added_formulae = []
       @modified_formula = []
       @steps = []
-      @tap = options[:tap]
+      @tap = options.fetch(:tap,  CoreFormulaRepository.instance)
       @repository = @tap.path
-      @skip_homebrew = options[:skip_homebrew]
+      @skip_homebrew = options.fetch(:skip_homebrew, false)
 
-      url_match = argument.match HOMEBREW_PULL_OR_COMMIT_URL_REGEX
-
-      git "rev-parse", "--verify", "-q", argument
-      if $?.success?
+      if quiet_system "git", "rev-parse", "--verify", argument
```

Comment:
From us but not from the `git` command itself. I guess I just see it being worth just being consistent with the old version. Don't care much either way though; you decide.